src/netex-convertor/netexGenerator.ts

```diff
@@ -263,19 +273,19 @@ const netexGenerator = (
 
         // Fare structure elements
         priceFareFrameToUpdate.tariffs.Tariff.fareStructureElements.FareStructureElement = getFareStructuresElements(
-            ticket,
+            ticket as PeriodTicket,
```

Comment:
Are these casts just temporary until we update the price fare frame function fully?